craft_store/base_client.py

```diff
@@ -32,6 +32,46 @@
 logger = logging.getLogger(__name__)
 
 
+def wrap_credentials(token_type: str, credentials: str) -> str:
```

Comment:
credentials at this point should still be a Union[Dict[str,str], str]

Reply:
this is how it would be used externally https://github.com/snapcore/snapd/pull/12065/files#diff-55b255a7f3873968742a5f29b8305075692c978bbaaabc7dd190ce5091112a41R179

Reply:
Nice catch, I had misinterpreted the spec. Updated.